packages/backend/src/open_payments/grant/service.test.ts

```diff
@@ -21,10 +21,6 @@ import {
 import { v4 as uuid } from 'uuid'
```

Comment:
The modifications here are all removing the `tenantId` parameter from grant creation calls